File: app/src/main/java/com/typ/muslim/integrations/telegram/models/TelegramChannel.java
/*
 * This app is developed by AHMED SLEEM
 *
 * Copyright (c) 2021.  TYP INC. All Rights Reserved
 */

package com.typ.muslim.integrations.telegram.models;

import com.typ.muslim.integrations.telegram.enums.TelegramChatType;

import java.util.List;

public class TelegramChannel extends TelegramChat{

	public TelegramChannel(int id, TelegramChatPhoto photo, String title, String description, TelegramChatPermissions botPermissions) {
		super(id, TelegramChatType.CHANNEL, photo, title, description, botPermissions);
	}
	// todo: idk what to do here
}

File: app/src/main/java/com/typ/muslim/features/telegram/models/TelegramChannel.kt
/*
 * This app is developed by AHMED SLEEM
 *
 * Copyright (c) 2021.  TYP INC. All Rights Reserved
 */
package com.typ.muslim.features.telegram.models

import com.typ.muslim.features.telegram.enums.TelegramChatType

class TelegramChannel(
    id: Int,
    photo: TelegramChatPhoto,
    title: String,
    description: String,
    botPermissions: TelegramChatPermissions
) : TelegramChat(
    id.toLong(),
    TelegramChatType.CHANNEL,
    photo,
    title,
    description,
    botPermissions
)

// todo: idk what to do here